sdk/tables/azure-data-tables/samples/sample_query_table.py

```diff
@@ -83,77 +83,42 @@ def sample_query_entities(self):
-        with TableClient.from_connection_string(self.connection_string, self.table_name) as table_client:
-            try:
+                print("Sample for querying entities withtout metadata:")
+                print("Entities with name: marker")
                 parameters = {"name": "marker"}
                 name_filter = "Name eq @name"
                 headers = {"Accept" : "application/json;odata=nometadata"}
                 queried_entities = table_client.query_entities(
                     query_filter=name_filter, select=["Brand", "Color"], parameters=parameters, headers=headers
                 )
-
                 for entity_chosen in queried_entities:
                     print(entity_chosen)
-
-            except HttpResponseError as e:
-                print(e.message)
-        # [END query_entities]
-
-    def sample_query_entities_multiple_params(self):
-        from azure.data.tables import TableClient
-        from azure.core.exceptions import HttpResponseError
-
-        print("Entities with name: marker and brand: Crayola")
-        # [START query_entities]
-        with TableClient.from_connection_string(self.connection_string, self.table_name) as table_client:
-            try:
+                
+                print("Sample for querying entities with multiple params:")
+                print("Entities with name: marker and brand: Crayola")
                 parameters = {"name": "marker", "brand": "Crayola"}
                 name_filter = "Name eq @name and Brand eq @brand"
                 queried_entities = table_client.query_entities(
                     query_filter=name_filter, select=["Brand", "Color"], parameters=parameters
                 )
-
                 for entity_chosen in queried_entities:
                     print(entity_chosen)
 
-            except HttpResponseError as e:
-                print(e.message)
-        # [END query_entities]
-
-    def sample_query_entities_values(self):
-        from azure.data.tables import TableClient
-        from azure.core.exceptions import HttpResponseError
-
-        print("Entities with 25 < Value < 50")
-        # [START query_entities]
-        with TableClient.from_connection_string(self.connection_string, self.table_name) as table_client:
-            try:
-                parameters = {"lower": 25, "upper": 50}
+                print("Sample for querying entities' values:")
+                print("Entities with 25 < Value < 50")
+                parameters = {"lower": 25, "upper": 50} # type: ignore
                 name_filter = "Value gt @lower and Value lt @upper"
                 queried_entities = table_client.query_entities(
                     query_filter=name_filter, select=["Value"], parameters=parameters
                 )
-
                 for entity_chosen in queried_entities:
                     print(entity_chosen)
-
             except HttpResponseError as e:
-                print(e.message)
+                pass
```

Comment:
Using `pass` on an HttpResponseError is probably not a good idea for a sample - we don't want people to copy and paste this by accident. I would probably print some kind of message then raise it.

Reply:
The exception raised here will be caught and print out in main(line131).

Reply:
No it wont - because this `pass` is suppressing it. It would need to be `raise`

Reply:
`raise` is what I meant, updated!